api/app/resources/endpoints.py

```diff
@@ -99,33 +90,33 @@ def get(self, page=1):
             page = 1
 
         if search:
-            query = Bucketlist.query.filter_by(
-                name=qs)
-            if query.count() == 0:
-                return {"message": "No search results"}
+            queries = Bucketlist.query.filter(Bucketlist.created_by == str(g.user.id))
+            bucketlists = []
+            for query in queries.all():
```

Comment:
**Code Quality: Nitpick**

Not a fan of program level filtering. There's a way to do this directly from the ORM. Much faster.